README.md

```diff
@@ -1,76 +1,208 @@
+[`jsondown`]: https://github.com/toolness/jsondown
+[`leveldown-basho`]: https://www.npmjs.com/package/leveldown-basho
+[`localstorage-down`]: https://github.com/No9/localstorage-down
+[`medeadown`]: https://github.com/kesla/medeadown
+[`mongodown`]: https://github.com/watson/mongodown
+[`mysqldown`]: https://github.com/kesla/mysqldown
+[`riakdown`]: https://github.com/nlf/riakdown
+[`sheet-down`]: https://github.com/jed/sheet-down
+[AWS DynamoDB]: http://aws.amazon.com/dynamodb
+[Google Sheets]: https://docs.google.com/spreadsheets
+[riakpbc]: https://github.com/nlf/riakpbc
+[medea]: https://github.com/argo/medea
+[Basho's LevelDB fork]: https://github.com/basho/leveldb
+[localStorage]: https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage
+
+[`encoding-down`]: https://github.com/level/encoding-down
+[`deferred-leveldown`]: https://github.com/Level/deferred-leveldown
+[`cachedown`]: https://github.com/mvayngrib/cachedown
+
+[`levelup-async-iterator`]: https://github.com/MeirionHughes/levelup-async-iterator
+[`level`]: https://github.com/level/level
+[`level-iterator-stream`]: https://github.com/level/iterator-stream
+[leveldb.org]: http://leveldb.org
+[level-badge]: https://img.shields.io/badge/-level-brightgreen.svg?colorA=000000&logo=data%3Aimage%2Fsvg%2Bxml%3Bbase64%2CPD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8%2BPHN2ZyB3aWR0aD0iMTg4cHgiIGhlaWdodD0iMjQycHgiIHZpZXdCb3g9IjAgMCAxODggMjQyIiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHhtbG5zOnNrZXRjaD0iaHR0cDovL3d3dy5ib2hlbWlhbmNvZGluZy5jb20vc2tldGNoL25zIj4gICAgPHRpdGxlPlVudGl0bGVkIDEzPC90aXRsZT4gICAgPGRlc2NyaXB0aW9uPkNyZWF0ZWQgd2l0aCBTa2V0Y2ggKGh0dHA6Ly93d3cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaCk8L2Rlc2NyaXB0aW9uPiAgICA8ZGVmcz48L2RlZnM%2BICAgIDxnIGlkPSJQYWdlLTEiIHN0cm9rZT0ibm9uZSIgc3Ryb2tlLXdpZHRoPSIxIiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIHNrZXRjaDp0eXBlPSJNU1BhZ2UiPiAgICAgICAgPHBhdGggZD0iTTk0LDU0IEwwLDEwMC40MTAwNDIgTDAsMTk0LjgwMzM0NyBMOTQsMjQyIEwxODgsMTk0LjgwMzM0NyBMMTg4LDEwMC40MTAwNDIgTDk0LDU0IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iI0Q0RUI5NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDM5IEwwLDg1LjQxMDA0MTggTDAsMTc5LjgwMzM0NyBMOTQsMjI3IEwxODgsMTc5LjgwMzM0NyBMMTg4LDg1LjQxMDA0MTggTDk0LDM5IFoiIGlkPSJSZWN0YW5nbGUtMSIgZmlsbD0iIzk2REM3NSIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICAgICAgPHBhdGggZD0iTTk0LDEgTDAsNDcuNDEwMDQxOCBMMCwxNDEuODAzMzQ3IEw5NCwxODkgTDE4OCwxNDEuODAzMzQ3IEwxODgsNDcuNDEwMDQxOCBMOTQsMSBaIiBpZD0iUmVjdGFuZ2xlLTEiIG9wYWNpdHk9IjAuNTUyODQwOTA5IiBmaWxsPSIjMzE3MzQyIiBza2V0Y2g6dHlwZT0iTVNTaGFwZUdyb3VwIj48L3BhdGg%2BICAgICAgICA8cGF0aCBkPSJNOTQsOTEuMTc3Nzc5MSBMMCw0NyBMMCwxNDEuNjY2NjY3IEw5NCwxODkgTDE4OCwxNDEuNjY2NjY3IEwxODgsNDcgTDk0LDkxLjE3Nzc3OTEgWiIgaWQ9IlJlY3RhbmdsZS0xIiBvcGFjaXR5PSIwLjU1Mjg0MDkwOSIgZmlsbD0iIzM0OTU0QyIgc2tldGNoOnR5cGU9Ik1TU2hhcGVHcm91cCI%2BPC9wYXRoPiAgICA8L2c%2BPC9zdmc%2B
```

Comment:
Put this badge SVG on leveldb.org?

Reply:
Perhaps needs to be the awesome badge here: see https://github.com/sindresorhus/awesome/blob/master/create-list.md

Reply:
IMO the awesome badge is rather ugly. Too much "bro" and doesn't match the standard badge style